src/fileformat/file_format/file_format.cpp

```diff
@@ -1091,6 +1091,24 @@ std::size_t FileFormat::getOverlaySize() const
 	return (realSize > declSize) ? realSize - declSize : 0;
 }
 
+/**
+ * Get overlay data entropy
+ * @param res Variable to store the result to
+ * @return @c true if entropy calculation succeeded, @false otherwise
+ */
+bool FileFormat::getOverlayEntropy(double &res) const
+{
+	const auto overlaySize = getOverlaySize();
+	const auto declSize = getDeclaredFileLength();
+	const auto bytes = getBytes();
```

Comment:
Beware that storing the result of
```cpp
const std::vector<unsigned char>& getBytes() const;
```
into
```cpp
const auto bytes = getBytes();
```
causes the vector to be copied, i.e. the type of `bytes` will be `const std::vector<unsigned char>`. I believe what you want is
```cpp
const auto &bytes = getBytes();
```
which will store a reference, i.e. the type of `bytes` will be `const std::vector<unsigned char> &`.